backend/src/schema.graphql

```diff
@@ -207,6 +210,10 @@ type Post {
   )
 }
 
+input _PostFilter {
```

Comment:
We don't have to redefine the `_PostFilter` on our side, that's done by the library `neo4j-graphql-js` when we augment the schema.

Reply:
@Tirokk the library will only add types and inputs etc. if they haven't been defined yet, so defining it on our side will effectively disable this behaviour.